pkg/instances/instances_test.go

```diff
@@ -3,3 +3,91 @@
 // Licensed under the MIT License.
 // ------------------------------------------------------------
 package instances
+
+import (
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/client-go/kubernetes/fake"
+)
+
+func newTestSimpleK8s(objects ...runtime.Object) instances {
+	client := instances{}
+	client.kubeClient = fake.NewSimpleClientset(objects...)
+	return client
+}
+
+func newDaprControlPlanePod(name string, appName string, creationTime time.Time, state v1.ContainerState, ready bool) *v1.Pod {
+	return &v1.Pod{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:        name,
+			Namespace:   "dapr-system",
+			Annotations: map[string]string{},
+			Labels: map[string]string{
+				"app": appName,
+			},
+			CreationTimestamp: metav1.Time{
+				Time: creationTime,
+			},
+		},
+		Status: v1.PodStatus{
+			ContainerStatuses: []v1.ContainerStatus{
+				{
+					State: state,
+					Ready: ready,
+				},
+			},
+		},
+		Spec: v1.PodSpec{
+			Containers: []v1.Container{
+				{
+					Image: name + ":0.0.1",
+				},
+			},
+		},
+	}
+}
+
+func TestControlPlaneServices(t *testing.T) {
+	controlPlaneServices := []struct {
+		name    string
+		appName string
+	}{
+		{"dapr-operator-67d7d7bb6c-7h96c", "dapr-operator"},
```

Comment:
one more change. please add dapr-dashboard and test it also

Reply:
it does not test anything new. the filter happens in the backend.